test/integration/bitswap_wo_routing_test.go

```diff
@@ -19,22 +17,15 @@ func TestBitswapWithoutRouting(t *testing.T) {
 	const numPeers = 4
 
 	// create network
-	mn, err := mocknet.FullMeshLinked(ctx, numPeers)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	peers := mn.Peers()
-	if len(peers) < numPeers {
-		t.Fatal(errors.New("test initialization error"))
-	}
-
-	// set the routing latency to infinity.
-	conf := testutil.LatencyConfig{RoutingLatency: (525600 * time.Minute)}
+	mn := mocknet.New(ctx)
```

Comment:
well, they arent linked until after the peers are added, and i'm adding the peers manually and calling the LinkAll method after adding peers, I think its still explicit.